docs/python/1.6/python-appapi-devguide-4.md

```diff
@@ -598,10 +598,10 @@ import union_source_functions
 
 def main():
     topo = Topology("union_source")
-   h = topo.source(union_source_functions.hello)
-   b = topo.source(union_source_functions.beautiful)
-   c = topo.source(union_source_functions.crazy)
-   w = topo.source(union_source_functions.world)
+    h = topo.source(union_source_functions.hello)
+    b = topo.source(union_source_functions.beautiful)
+    c = topo.source(union_source_functions.crazy)
+    w = topo.source(union_source_functions.world)
     streamSet = {b, c, w}
```

Comment:
We should fix `streamSet` since that is not python style, more typical would be:

```
hwu = h.union({b, c, w})
```